python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -959,3 +841,4 @@ def session_to_info(self, session):
         except Exception, err:
             logger.debug( "Session Info Exception: %s" % err)
             return session
+
```

Comment:
W298 according to pep8 says it should exist and I've editors which expect it to be there as good practice. Not functionally needed but recommended.